### PR TITLE
[Improve] Add agent settings for lark alarm.

### PR DESCRIPTION
**Pull request by chengyuan1029**
Add agent settings for lark alarm.#2364

### CODE REVIEW

**Comment by wolfboys** [comment]
Thanks for your contribution, please submit an issue first, describing the background and motivation clearly

**Comment by chengyuan1029** [comment]
> Thanks for your contribution, please submit an issue first, describing the background and motivation clearly

ok

File: streampark-console/streampark-console-service/src/main/resources/application.yml [hunk 1]
```diff
@@ -131,3 +131,9 @@ ldap:
     email-attribute: mail
     # action when ldap user is not exist (supported types: CREATE,DENY)
     not-exist-action: CREATE
+
+alert:
```
Comment (wolfboys) [inline streampark-console/streampark-console-service/src/main/resources/application.yml]: I suggest parameter  change to: 
```
streampark:
  proxy:
    lark-url:
```

Comment (chengyuan1029) [reply streampark-console/streampark-console-service/src/main/resources/application.yml]: ok

**Review by wolfboys — APPROVED** [review]
LGTM

**Pull request merged**